pytest_plugins/rerun_rp/rerun_rp.py

```diff
@@ -126,7 +126,7 @@ def pytest_collection_modifyitems(items, config):
     test_args = {}
     if fail_args:
         test_args['status'] = 'failed'
-        if not fail_args == ['all']:
+        if not fail_args == 'all':
```

Comment:
`if fail_args != 'all'` would probably be a bit better syntax here, but this works.